markdown/bitburner.corporation.createcorporation.md

```diff
@@ -29,9 +29,3 @@ true if created and false if not
 
 RAM cost: 20 GB
 
-This function throws an error if:
```

Comment:
Please leave this, it's good/important info. You could probably take out the but about getBitNodeMultipliers now if you want.

Reply:
When I put it back, I also fix it. The old "Try to self-fund outside BitNode 3." is wrong. It's "use seed money", not "self-fund". I made that mistake in #1587.